README.md

```diff
@@ -116,6 +116,8 @@ LIST OPTIONS
         show branches by date
     -C, --contributors
         see a list of everyone who contributed to the repo
+    -n, --new-contributors
```

Comment:
Thanks for adding how to use the new functionality here and in the source! For completeness, let's add this to the manpages, too. That way, when someone calls `man` on `git-quick-stats`, they can see this appear in there, too.